config/default/default.vcl

```diff
@@ -0,0 +1,192 @@
+# Based on https://fourkitchens.atlassian.net/wiki/display/TECH/Configure+Varnish+3+for+Drupal+7
```

Comment:
Why do we need this, if thee are configs per version anyway? If individual version configs are identical, then we should use a single one instead.